### DISCUSSION TITLE
Location without Google Play Services

### DISCUSSION THREAD

**Discussion by marcocastignoli** [ionic-team/capacitor#3528] [Q&A] [answered]
I'm trying to get the location from a device without Google Play Services, is it possibile?

**Comment by jcesarmobile** [comment, marked as answer]
you can use the WebView geolocation [`navigator.geolocation`](https://developer.mozilla.org/en-US/docs/Web/API/Navigator/geolocation) instead of the Geolocation plugin, but it won't work on iOS

**Reply by marcocastignoli** [reply]
thanks
